README.md

```diff
@@ -147,13 +147,27 @@ These instructions will get you a copy of the project up and running on your loc
 - `make`
 - `git`
 
-## Getting Started
+## Using `make`
 
 We've made the startup process simple by using a Makefile for all common workflows.
 
-To see what Makefile commands are available, in your terminal shell run `make` or `make help`. It will list all the available Makefile commands with their descriptions, such as
+To see what Makefile commands are available, in your terminal shell run `make` or `make help`. It will list all the available Makefile 
+commands with their descriptions.
```

Comment:
Is this a formatter at work?

Reply:
I'm not sure where that came from tbh... I'll remove the line break.